lib/packer-sync.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+var zlib = require('zlib');
+var constants = require('./constants');
+var Packer = require('./packer');
+
+module.exports = function(metaData, opt) {
+  var options = opt || {};
+  options.deflateChunkSize = options.deflateChunkSize || 32 * 1024;
```

Comment:
it would be good if these defaults could be shared between the two implementations - perhaps create a getOptions on packer and use it in both places (including the throw error for things not supported)